index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+   
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
```

Comment:
Kindly add the close tag to your main tag and make it wrap the main content of your work

Reply:
Kindly consider explaining to the student why you are requesting for such a change. :+1:  It may be clear to you but not the student, hence you may end up confusing him/her the more. :confused: